db/db_options_test.cc

```diff
@@ -178,6 +178,23 @@ TEST_F(DBOptionsTest, EnableAutoCompactionAndTriggerStall) {
   }
 }
 
+TEST_F(DBOptionsTest, EnableAutoCompactionShouldTriggerCompaction) {
+  Options options;
+  options.create_if_missing = true;
+  options.level0_file_num_compaction_trigger = 1000;
+  Reopen(options);
+  Random rnd(370);
+  for (int i = 0; i < 3; i++) {
+    GenerateNewRandomFile(&rnd);
+    Flush();
```

Comment:
Every use case, tune the memtable size carefully to make sure a flush is triggered.